proto/tokenfactory/query.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package seiprotocol.seichain.tokenfactory;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "tokenfactory/authorityMetadata.proto"; 
+import "tokenfactory/params.proto"; 
+
+option go_package = "github.com/sei-protocol/sei-chain/x/tokenfactory/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // Params defines a gRPC query method that returns the tokenfactory module's
+  // parameters.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/osmosis/tokenfactory/v1beta1/params";
```

Comment:
btw these paths have `osmosis` in them which we should fix, probably best to just find all "osmosis" and correct them to sei or something

Reply:
Updated